src/android/device.rs

```diff
@@ -176,6 +397,83 @@ impl<'a> Device<'a> {
         Ok(())
     }
 
+    fn clean_apks(&self, config: &Config, profile: Profile) -> Result<(), ApksBuildError> {
+        let flavor = self.target.arch;
+        let apks_path = Self::apks_path(config, profile, flavor);
+        bossy::Command::impure_parse("rm -f")
+            .with_arg(
+                apks_path
+                    .to_str()
```

Comment:
`.with_arg` takes anything that impls `AsRef<OsStr>`, so you don't need to convert to a string here (or in any fs functions)